statesync/blocks.go

```diff
@@ -0,0 +1,263 @@
+package statesync
```

Comment:
nit: I think we should rename this file to `block_queue.go`.

Reply:
ok that makes sense to me as well